suite-native/formatters/src/components/FiatAmountFormatter.tsx

```diff
@@ -9,7 +9,7 @@ import { FormatterProps } from '../types';
 import { EmptyAmountText } from './EmptyAmountText';
 import { AmountText } from './AmountText';
 
-type CryptoToFiatAmountFormatterProps = FormatterProps<string | number | null> &
+type CryptoToFiatAmountFormatterProps = FormatterProps<string | null> &
```

Comment:
```suggestion
type FiatAmountFormatterProps = FormatterProps<string | null> &
```

I know it is not your mistake, but fix it please. It needs to be changed also on line 23 to apply correctly named props.